src/test/java/pages/GoogleHomePage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class GoogleHomePage extends AbstractPage {
+
+    private static final String HOMEPAGE_URL = "https://google.com";
```

Comment:
GOOGLE_URL

Reply:
Refactored, thank you.
Please see in the following commit.